protocols/imap/src/main/java/org/apache/james/imap/processor/base/SelectedMailboxImpl.java

```diff
@@ -411,7 +411,7 @@ public synchronized void event(Event event) {
 
```

Comment:
Commit name is s/getApplicableFlag/getApplicableFlags/ and this diff is exactly about that. But maybe I am missing something